modules/ducktests/tests/ignitetest/tests/smoke_test.py

```diff
@@ -87,3 +87,28 @@ def test_zk_start_stop(self):
         zookeeper = ZookeeperService(self.test_context, num_nodes=3)
         zookeeper.start()
         zookeeper.stop()
+
+    @cluster(num_nodes=4)
+    @ignite_versions(str(DEV_BRANCH))
+    def test_ignite_clock_sync(self, ignite_version):
```

Comment:
It's not a smoke test of services. It checks that environment is configured correctly. So it's named as self tests. Actually there is a one test with the same goal - assertion_test. Let's rename it to smth like self_tests.py and move this test to the file.

Reply:
Fixed

Reply:
Let's remove this test from file.